.pre-commit-config.yaml

```diff
@@ -35,7 +35,8 @@ repos:
     rev: v0.4.2
     hooks:
     -   id: style-files
-        args: [--style_pkg=styler, --style_fun=tidyverse_style]
+        args: [--style_pkg=styler, --style_fun=tidyverse_style,
+               --cache-root=styler-perm]
```

Comment:
Is it worth updating what is in here on the cdcent template repo?

Reply:
Arguably but this isn't as critical as the use of `actions/cache`